src/style.css

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+}
+
+.rating {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  display: flex;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
```suggestion
```

you can remove `background-repeat` and `background-position`, because it's already defined in `.stars__star`

Reply:
you're right! looks definitely better without duplicates and still works :)